### ISSUE TITLE
Get this working on asusmerlin-wrt

### ISSUE THREAD

**Issue by geligelu** [clowwindy/ShadowVPN#162]
Anybody can help point me in the direction of getting ShadowVPN on ASUSMerlin-WRT?

**Comment by GuBo88** [comment]
I"m also interested in this.

**Comment by McContax** [comment]
@geligelu Actually, The main developer had abandoned this project.Few months ago @aa65535 still worked on the openwrt.But I didn't see anything about him these weeks.

**Comment by geligelu** [comment]
@McContax  - that's a shame. I saw that ShadowVPN is the fastest available in China and I really wanted to try it. Although - there's such little documentation.

Did you need your own server to run it? I mean it's all good getting it set up on your router - but you need a VPN server to connect to, to make it work.

**Comment by McContax** [comment]
@geligelu Yes. It's a really good project. But the Chinese government didn't allow this. So the main developer had to give up. I forked the code before the author delete it. I think It will be needed someday.
Now I run a shadowvpn-server(It also contain in this project) on a VPS. It's a good way to improve the game connection by using this project.Though it is not really perfect. Actually it's still a VPN so there will be server-side and client-side.After setting up a client-side on rounter.U need to set up a server-side on a server or vps.
